.teamcity/_self/builds/WPComPlugins_EditorToolkit.kt

```diff
@@ -102,36 +95,52 @@ object WPComPlugins_EditorToolKit : BuildType({
 				cd apps/editing-toolkit
 				yarn build
 
-				echo
-				prev_release_build_num=${'$'}(grep build_number ../../etk-release-build/build_meta.txt | sed -e "s/build_number=//")
-				rm ../../etk-release-build/build_meta.txt # Adds a source of randomness which we don't want for comparison.
-				echo "Previous tagged trunk build: ${'$'}prev_release_build_num"
-
 				# Update plugin version in the plugin file and readme.txt.
-				# Note: we also update the previous release build to the same version to restore idempotence
+				# We also update the previous trunk version to match, so that
+				# we can diff the old and new versions without random data.
 				sed -i -e "/^\s\* Version:/c\ * Version: %build.number%" -e "/^define( 'A8C_ETK_PLUGIN_VERSION'/c\define( 'A8C_ETK_PLUGIN_VERSION', '%build.number%' );" ./editing-toolkit-plugin/full-site-editing-plugin.php ../../etk-release-build/full-site-editing-plugin.php
 				sed -i -e "/^Stable tag:\s/c\Stable tag: %build.number%" ./editing-toolkit-plugin/readme.txt ../../etk-release-build/readme.txt
+			"""
+			executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
```

Comment:
Why do we want to run this script (and the next one) even in if the previous fails?

Reply:
Good question. I think these were artifacts from the original implementation, so I left them in. I've removed "run_on_failure" from every step in this build.